install/hst-install-debian.sh

```diff
@@ -2039,6 +2039,7 @@ write_config_value "POLICY_USER_VIEW_SUSPENDED" "no"
 write_config_value "POLICY_USER_VIEW_LOGS" "yes"
 write_config_value "POLICY_USER_EDIT_WEB_TEMPLATES" "true"
 write_config_value "POLICY_USER_EDIT_DNS_TEMPLATES" "yes"
+write_config_value "API_V2" "0"
```

Comment:
On Initial install it doesn't create the /usr/local/hestia/data/api key folder + copies the preset(s) to the folder

Reply:
We should probally enable it for admin user only on default if api is enabled